core-model/src/commonMain/resources/MR/zh-rCN/strings.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<resources>
+    <!-- Navigation Drawer -->
+    <string name="title_sessions">动态消息</string>
+    <string name="title_about">关于DroidKaigi</string>
+    <string name="title_announcement">通知</string>
+    <string name="title_map">楼面导轨</string>
+    <string name="title_sponsors">赞助者</string>
+    <string name="title_contributors">Contributor</string>
+    <string name="title_setting">设置</string>
+
+    <!-- About -->
+    <string name="about_top_app_bar_title">DroidKaigiとは</string>
```

Comment:
I have a question! If we don't have these strings, it will fail to build?

Reply:
Our lint would prevent our build on the CI to pass.

Reply:
I see. If I use it in production. We should consider it. 👀 
https://github.com/icerockdev/moko-resources/issues/331